client/post-editor/post-editor.jsx

```diff
@@ -360,12 +360,7 @@ export const PostEditor = createReactClass( {
 								) }
 							</div>
 							<div className="post-editor__inner-content">
-								<FeaturedImage
-									site={ site }
-									post={ this.state.post }
-									maxWidth={ 1462 }
-									hasDropZone={ true }
-								/>
+								<FeaturedImage maxWidth={ 1462 } hasDropZone />
```

Comment:
Total aside that this magic number seems quite odd. 🤷‍♀️  Doing a quick search in history I didn't find a reason why it's this value.

Reply:
It's been this way since the initial OSS commit. I also don't know why this precise value :smile: